src/mobile/android/build.gradle

```diff
@@ -48,6 +48,13 @@ subprojects {
             }
         }
     }
+    if (project.name.contains('react-native-modal-translucent')) {
+            buildscript {
+                repositories {
+                maven { url "https://dl.bintray.com/android/android-tools/"  }
```

Comment:
Shouldn't this be added on line 27 of this file?

Reply:
Following this solution https://github.com/listenzz/react-native-modal-translucent/issues/12

Reply:
Ahh, I see 👍 I thought this was something different